index.js

```diff
@@ -26,14 +26,14 @@ const getFilteredChangedFiles = (changedFiles, fileExtensions, excludePaths) =>
 };
 
 const getApproxMaxSymbols = () => {
-    const maxTokens = 8192;
+    const maxTokens = 70000;
     const maxSymbols = maxTokens * 4;
```

Comment:
The change in `maxTokens` from `8192` to `70000` represents an increase by approximately 8.5 times. This change can potentially lead to significantly larger input sizes for functions that rely on `getApproxMaxSymbols`. Please ensure that all systems (including the OpenAI API if used) can handle this increased input size without any issues, such as rate limits, memory constraints, or performance degradation.